packages/ui/src/modal/buy/BuyModalRenderer.tsx

```diff
@@ -250,6 +250,7 @@ export const BuyModalRenderer: FC<Props> = ({
       paymentCurrency: EnhancedCurrency | undefined,
       paymentTokens: EnhancedCurrency[]
     ) => {
+      console.log(paymentTokens)
```

Comment:
remove log

Reply:
thanks @ted-palmer can always count on you